### PR TITLE
BD-1247: Link Aliasing 101

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-1247**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

### CODE REVIEW

**Review by gsinghAppboy — DISMISSED** [review]
lgtm ty!

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## What to expect
+
+When Link Aliasing is enabled, expect the following behaviors to occur:
+1. All recognized links in an email are decorated with an additional query parameter `lid={{"placeholder"}}`, where `{{"placeholder"}}` is a unique Liquid-generated alphanumeric value. This occurs when the client either clicks the **Link Management** tab in the composer or clicks **Done** to navigate back to the message workflow.
+2. New Content Blocks will have their links modified by appending a `lid={{"placeholder"}}` to each link, where applicable. The `{{"placeholder"}}` value is resolved when inserted into an email message variant only. If the HTML content block is utilized in other channels, such as in-app messages, the `lid` query parameter will still be appended.
+	- Content Blocks that were created before Link Aliasing enablement will not have their links recognized byu the feature. Customers are encouraged to duplicate and create new Content Blocks that include the Link Aliasing query parameters.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:19]: small typo!

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+{% alert note %}
+Clients can also update their existing content block. However, only a maximum of 50 messages referencing that content block will be updated for Link Aliasing purposes.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:22]: Content Block capitalization

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## What use cases does this potentially unlock?
+
+Clients can now distinguish between the same URL present in their email message by providing unique link alias names to each one (for example, "main1" and "main2", which both direct to https://www.braze.com). Clients can now retarget individual aliases that are clicked in segmentation.
+
+{% alert note %}
+There is a maximum of 100 aliases allowed for retargeting purposes at this time.
+{% endalert %}
+
+Clients can create action-based triggers based on an alias that is clicked since as there's no limit here since action-based is triggered "in the moment" we receive a click event.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:33]: since as -> as

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Can you show me what my links will look like?
+Braze looks for the anchor tag in the HTML to identify the link (`<a href=””> Link Text </a>`). Braze will then append based on the presence of a question  mark(?), which indicates that query parameters can be appended.  
+
+Here are a few examples where Braze can safely append the `lid` query parameter:
+
+| Link in Email Body | Link with Aliasing| Logic |
+|---|---|---|
+| https://www.braze.com | https://www.braze.com?lid=slfdldtqdhdk | Braze inserts a question mark (?) and adds the first query parameter into the URL. |
+| https://www.braze.com?utm_campaign=retention&utm_source=email | https://www.braze.com?utm_campaign=retention&utm_source=email&lid=0goty30mviyz | Braze detects other query parameters and appends "lid=" to the end of the URL. |
+| `<a href="{{custom_attribute.{product_url}}?">` | `<a href=”{{custom_attribute.{product_url}}?lid=ac7a548g5kl7”>` | Braze recognizes that this is a URL and already has a question mark (?) present. Then, it appends the lid query parameter after the question mark. |
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:49]: lid -> `lid`

Comment (KellieHawks) [reply _docs/_hidden/private_betas/link_aliasing101.md:49]: same with bottom row of table

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+---
+nav_title: Link Aliasing 101
+permalink: /link_aliasing_101/
+description: "This article describes what to expect when Link Aliasing is enabled, such as impacts on your workflow and what your links will look like."
+hidden: true
+---
+
+# Link Aliasing 101
+
+## What is link aliasing?
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:10]: Capitalization here for Link Aliasing?

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Hey @lydia-xie, looks great! Left a few suggestions for small edits.

File: _docs/_hidden/private_betas/link_aliasing101.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+Link Aliasing is a new feature that allows marketers to create recognizable, user-generated names to identify links sent in email messages from Braze. These user-generated names are then made available for segmentation retargeting and action-based triggering, and in the reporting display of click information.
+
+## What to expect
+
+When Link Aliasing is enabled, expect the following behaviors to occur:
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/link_aliasing101.md:16]: Hey @lydia-xie one more thing, can you make sure the {{}} are showing up in your doc. If they aren't you will have to encapsulate the text individually or the entire section with {%raw%} {%endraw%}

**Review by KellieHawks — APPROVED** [review]
Look great! Thanks for making those edits.

**Pull request merged**